### PR TITLE
Add Max Speed per drive information

### PR DESCRIPTION
**Pull request by jonferreira**

### CODE REVIEW

File: grafana/dashboards/trip.json [hunk 1]
```diff
@@ -1553,7 +1577,7 @@
...
...N geofences start_geofence ON start_geofence_id = start_geofence.id\n  LEFT JOIN geofences end_geofence ON end_geofence_id = end_geofence.id\n  LEFT JOIN cars car ON car.id = drives.car_id\n  WHERE $__timeFilter(start_date) AND drives.car_id = $car_id\n  GROUP BY drives.id\n)\nSELECT\n    start_date_ts,\n    end_date_ts,\n    car_id,\n    start_path,\n    end_path,\n    duration_str,\n    drive_id,\n    -- Columns\n    start_date,\n    start_address,\n    end_address,\n    duration_min,\n    convert_km(distance::numeric, '$length_unit') AS distance_$length_unit,\n    convert_km(speed.speed_max::numeric, '$length_unit') AS speed_max_${length_unit}h,\n    start_battery_level as \"% Start\",\n    end_battery_level as \"% End\",\n    CASE WHEN is_sufficiently_precise THEN range_diff * car_efficiency / distance * 1000 * CASE WHEN '$length_unit' = 'km' THEN 1\n                                                                                                WHEN '$length_unit' = 'mi' THEN 1.60934\n                                                                                           END\n    END AS consumption_kWh_$length_unit\nFROM data\nLEFT JOIN speed ON speed.id = data.drive_id\n;",
```
Comment (adriankumpf) [inline grafana/dashboards/trip.json:1580]: Drives already have a `speex_max` column

File: grafana/dashboards/trip.json [hunk 2]
```diff
@@ -1517,6 +1517,30 @@
...
+              "options": "speed_max_kmh"
+            },
+            "properties": [
+              {
+                "id": "displayName",
+                "value": "Max. Speed"
+              },
+              {
+                "id": "unit",
+                "value": "lengthkm"
```
Comment (adriankumpf) [inline grafana/dashboards/trip.json:1533]: This should be `km/h`

File: grafana/dashboards/trip.json [hunk 3]
```diff
@@ -1517,6 +1517,30 @@
                 "value": true
               }
             ]
+          },
+          {
+            "matcher": {
+              "id": "byName",
+              "options": "speed_max_kmh"
```
Comment (adriankumpf) [inline grafana/dashboards/trip.json:1524]: Another matcher would have to be added for mph :)

**Review by adriankumpf — CHANGES_REQUESTED** [review]
Thanks for the PR!

The max speed is not a good fit for the Trip dashboard as it is meant to be display only the most important drive related information. It could be added to the Drives dashboard though.

**Comment by Dulanic** [comment]
Closing, if want to add, need to rebase and adjust per feedback. If someone else wants to take over, resubmit. Agree that this does not belong in trip.

**Pull request closed without merging**